ouroboros-consensus/src/Ouroboros/Storage/ImmutableDB/Impl/Iterator.hs

```diff
@@ -305,7 +312,7 @@ streamImpl dbEnv blocksOrHeaders mbStart mbEnd =
 
         -- Use the secondary index entry to determine whether the slot + hash
         -- correspond to an EBB or a regular block.
-        return $ (secondaryOffset,) $ (, hash) $
+        return $ (secondaryOffset,) $ (WithHash hash) $
```

Comment:
Unnecessary parens?

Reply:
Thanks for catching these :pray:. It used to be a tuple, so I forgot to remove the parentheses :nerd_face: